src/locales/en-US/list.ftl

```diff
@@ -82,8 +82,8 @@ item-summary-copy-password = Copy Password
 # Placeholder and accessibility label used in the field to filter existing
 # entries, i.e. search in the product database
 item-filter =
-  .placeholder = Search { -product-short-name }
-  .aria-label = Search { -product-short-name }
+  .placeholder = Search Logins
+  .aria-label = Search Logins
 
 add-item-button =
```

Comment:
`.title` attribute should be removed now that it's no longer being used. Just "New Login" for add-item-button should work fine

Reply:
Hmm, that's a legit bug. The `title` attr should still be used. I might have rebased this incorrectly...

Reply:
the latest on `desktop-ux` should _**not**_ use `title` attribute.

Reply:
oh, rebased and I see the .title was indeed removed. thanks :Beers:

Reply:
sorry for the terseness; as the button went from "wordless glyph" to "real words", the title attribute became unnecessary.

Reply:
np!